content/en/docs/ibm/deploy/install-kubeflow.md

```diff
@@ -315,3 +321,28 @@ Then visit `https://<kubeflow-FQDN>/`, it should redirect you to AppID for authe
 ## Additional information
 
 You can find general information about Kubeflow configuration in the guide to [configuring Kubeflow with kfctl and kustomize](/docs/other-guides/kustomize/).
+
+## Troubleshooting
+
+### Expose the Kubeflow endpoint as LoadBalancer
+
+By default, the Kubeflow deployment on IBM Cloud only exposes the endpoint as [NodePort](https://kubernetes.io/docs/concepts/services-networking/service/#nodeport) 31380. If you want to expose the endpoint as [LoadBalancer](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer), run:
+
+```shell
+kubectl patch svc istio-ingressgateway -n istio-system -p '{"spec": {"type": "LoadBalancer"}}'
+```
+
+Then, the LoadBalancer is located at the **EXTERNAL_IP** column when you run the below command:
+
+```shell
+kubectl get svc istio-ingressgateway -n istio-system
+```
+
+### Uninstall Kubeflow
+
+Run the below commands to uninstall Kubeflow and clean up all the local cache. Note that `${CONFIG_FILE}` is the [kfdef](https://www.kubeflow.org/docs/other-guides/kustomize/#specifying-a-configuration-file-when-initializing-your-deployment) for deploying Kubeflow.
```

Comment:
Usually, using "directional" language like "above"/"below" is not great for accessibility and most of our use cases are simple, so we should probably stick to anything that's similar to "following" or "before". I'd leave it up to you though.

```suggestion
To uninstall Kubeflow and clean up all the local cache, run the following command:
```

(moving the 2nd sentence towards the end)